E1_coldflow/brain_utils.h

```diff
@@ -16,15 +16,37 @@
 #include <string>
 #include <SdFat.h>
 #include <TimeLib.h>
+#include <vector>
 
 String make_packet(struct sensorInfo sensor);
 uint16_t Fletcher16(uint8_t *data, int count);
 void chooseValveById(int id, struct valveInfo *valve);
 
 const int numCryoTherms = 2;
-int cryoThermAddrs[numCryoTherms] = {0x60, 0x67}; //the second one is 6A or 6B, not sure which for Addr pin set to 1/2
+int cryoThermAddrs[numCryoTherms] = {0x60, 0x67};
 _themotype cryoTypes[numCryoTherms] = {MCP9600_TYPE_J, MCP9600_TYPE_T};
```

Comment:
missing an r - typo? should be `_thermotype`

Reply:
unfortunately not lol. the author of that lib has the typo

Reply:
but any comments on the general structure implementation?